onyo/commands/tree.py

```diff
@@ -17,10 +19,48 @@ def build_tree_cmd(directory):
     return "tree \"" + directory + "\""
 
 
+def check_sources(sources):
```

Comment:
This is relevant to all commands, and we already talked about this when we synced on video, but I just want to register my objections to the vague-ness of the name, and that it returns a modified copy of the sources, rather than a boolean.